eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovCertSpec.hs

```diff
@@ -107,6 +107,52 @@ spec = do
       -- Confirm that after registering a hot key, they are active
       _hotKey <- registerCommitteeHotKey cc
       ccShouldNotBeResigned cc
+  describe "Removing CC with UpdateCommittee" $ do
+    it "Non registered" $ do
+      (drepCred, _, _) <- setupSingleDRep 1_000_000
+      (spoC, _, _) <- setupPoolWithStake $ Coin 42_000_000
+      passNEpochs 2
+      initialCommittee <- getCommitteeMembers
+      logToExpr initialCommittee
+      initialCommittee `shouldSatisfy` not . Set.null
+      removeCCGaid <- submitUpdateCommittee Nothing initialCommittee mempty (1 %! 2)
+      submitYesVote_ (DRepVoter drepCred) removeCCGaid
+      submitYesVote_ (StakePoolVoter spoC) removeCCGaid
+      passNEpochs 2
+      finalCommittee <- getCommitteeMembers
+      logToExpr finalCommittee
+      finalCommittee `shouldSatisfy` Set.null
+    it "Registered" $ do
+      (drepCred, _, _) <- setupSingleDRep 1_000_000
+      (spoC, _, _) <- setupPoolWithStake $ Coin 42_000_000
+      passNEpochs 2
+      initialCommittee <- getCommitteeMembers
+      logToExpr initialCommittee
+      initialCommittee `shouldSatisfy` not . Set.null
+      forM_ (Set.toList initialCommittee) $ \kh -> do
+        ccHotCred <- KeyHashObj <$> freshKeyHash
+        submitTx_ $
+          mkBasicTx mkBasicTxBody
+            & bodyTxL . certsTxBodyL
+              .~ SSeq.singleton (AuthCommitteeHotKeyTxCert kh ccHotCred)
+      newCommittee <- arbitrary
+      let arbitrarySubset s = do
+            n <- choose (0, Set.size s)
+            subsetGo s mempty n
+          subsetGo pickFrom result n
+            | n == 0 = pure result
+            | otherwise = do
+                picked <- elements $ Set.toList pickFrom
+                subsetGo (Set.delete picked pickFrom) (Set.insert picked result) (n - 1)
+      initialSubCommittee <- arbitrarySubset initialCommittee
```

Comment:
We already have a function like this implemented: [`uniformSubSet`](https://github.com/IntersectMBO/cardano-ledger/blob/0ae0b17ba3dd04bc7c3f0985e80c39a252fdd156/libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Core/Arbitrary.hs#L637C1-L637C14)
```suggestion
      initialSubCommittee <- askStatefulGen >>= uniformSubSet Nothing initialCommittee 
```

See examples on usage of [`askStatefulGen` here](https://github.com/IntersectMBO/cardano-ledger/blob/0ae0b17ba3dd04bc7c3f0985e80c39a252fdd156/libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Imp/Common.hs#L418C24-L418C42)